src/components/calculator/calculator.js

```diff
@@ -25,28 +26,31 @@ const buttons = [
   '=',
 ];
 
+const initialObject = { total: 0, next: null, operation: null };
 const Calculator = () => {
-  const [calculatorInputs, setCalculatorInputs] = useState('0');
+  // const [calculatorInputs, setCalculatorInputs] = useState('0');
```

Comment:
- - You have done great so far. :+1:  Your calculator app works great but there are a few tasks you need to complete before moving to the next milestone. :+1: 
- [ ] It is against the best practices to leave unused code as a comment in your codebase, I'll suggest you remove this as it makes your codebase a bit unprofessional. :smile: :wave: